demos/python_demos/common/models/retinaface.py

```diff
@@ -19,20 +19,24 @@
 import numpy as np
 
 from .model import Model
-from .utils import DetectionWithLandmarks, resize_image
+from .utils import DetectionWithLandmarks, Detection, resize_image
 
 
 class RetinaFace(Model):
     def __init__(self, ie, model_path, threshold=0.5, mask_threshold=0.5):
         super().__init__(ie, model_path)
 
         assert len(self.net.input_info) == 1, "Expected 1 input blob"
-        assert len(self.net.outputs) == 12 or len(self.net.outputs) == 9, "Expected 12 or 9 output blobs"
+        expected_outputs_count = (6, 9, 12)
+        assert len(self.net.outputs) in expected_outputs_count, "Expected {} output blobs".format(
```

Comment:
This message ends up being "Expected 6, 9, 12 output blobs", which doesn't make sense. It should be "6, 9 or 12".

Reply:
Done